content/en/docs/Getting Started/tasks.md

```diff
@@ -0,0 +1,178 @@
+
+## Create your Kubernetes cluster
+
+Create a cluster
+
+```bash
+minikube start
+```
+
+The process takes a few seconds, you see an output symilar to the following,
+depending on the [minikube driver](https://minikube.sigs.k8s.io/docs/drivers/)
+that you are using:
+
+<pre>
+😄  minikube v1.25.1
+✨  Using the docker driver based on existing profile
+👍  Starting control plane node minikube in cluster minikube
+🚜  Pulling base image ...
+🔄  Restarting existing docker container for "minikube" ...
+🐳  Preparing Kubernetes v1.23.1 on Docker 20.10.12 ...
+    ▪ kubelet.housekeeping-interval=5m
+    ▪ Generating certificates and keys ...
+    ▪ Booting up control plane ...
+    ▪ Configuring RBAC rules ...
+🔎  Verifying Kubernetes components...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+</pre>
+
+You can check that the cluster was successfully created with `kubectl`:
+
+```bash
+kubectl cluster-info
+```
+
+The output confirms that Kubernetes is running:
+
+<pre>
+Kubernetes control plane is running at https://127.0.0.1:39509
+CoreDNS is running at
+https://127.0.0.1:39509/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+</pre>
+
+## Install Tekton Pipelines
+
+1. To install the latest version of Tekton Pipelines, use `kubectl`:
+
+   ```bash
+   kubectl apply --filename \
+   https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+1. Monitor the installation:
+
+   ```bash
+   kubectl get pods --namespace tekton-pipelines --watch
+   ```
+
+When all components show `Running` under the `STATUS` column the installation
+is complete.
+
+Hit *Ctrl + C* to stop monitoring.
+
+## Create and run a basic task
+
+A **task**, represented in the API as an object of kind `Task`, defines a
+series of **steps** that run sequentially to perform logic that the task
+requires. Every task runs as a pod on your Kubernetes cluster, with each step
+running in its own container.
+
+1. To create a task, open your favorite editor and create a file named
+   `hello-world.yaml` with the following content:
+
+   ```yaml
+   apiVersion: tekton.dev/v1beta1
+   kind: Task
+   metadata:
+     name: hello
+   spec:
+     steps:
+       - name: echo
+         image: alpine
+         script: |
+            echo "Hello World"
+   ```
+
+1. Apply the changes your cluster:
+
+   ```bash
+   kubectl apply --filename hello-world.yaml
+   ```
+
+     The output confirms that the task was completed successfully.
+
+     ```bash
+     task.tekton.dev/hello created
+     ```
+
+1. To run this task, you must instantiate it using `TaskRun`. Create another
+   file named `hello-world-run.yaml` with the following content:
+
+   ```yaml
```

Comment:
similar indentation issue with this block when rendered

Reply:
Fixed